metrics/deploy/ceph-configmap.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# This is a sample configmap that helps define a Ceph configuration.
+
+# Sample ceph.conf available at
+# https://github.com/ceph/ceph/blob/master/src/sample.ceph.conf Detailed
+# documentation is available at
+# https://docs.ceph.com/en/latest/rados/configuration/ceph-conf/
+apiVersion: v1
+kind: ConfigMap
+data:
+  ceph.conf: |
```

Comment:
instead of creating a configmap create these files in the code so we don't need to worry about removing configmap later?

Reply:
Sounds good. Will do.

Reply:
@Madhu-1 Doesn't Rook or Ceph CSI create this config?
Exporter deployment needs this config to be available before starting container since it needs to be mounted.

Reply:
https://github.com/ceph/ceph-csi/blob/devel/internal/util/cephconf.go cephcsi creates it internally.